fly.io/cdn-2024-01-26/fly.toml

```diff
@@ -0,0 +1,43 @@
+# Full app config reference: https://fly.io/docs/reference/configuration/
+app = "cdn-2024-01-26"
+# Closest to James
+primary_region = "sjc"
+# Secondary region will be "lhr", closest to Gerhard
+
+kill_signal = "SIGTERM"
+kill_timeout = 30
+
+[env]
+VARNISH_SIZE="500M"
```

Comment:
I think this `VARNISH_SIZE="500M"` environment variable tells Varnish that it can use 500MB of RAM for it's service, but down below you have the VM set to only have `256MB` of RAM, so this may result in Varnish trying to use more memory than the system has available.
https://www.varnish-software.com/developers/tutorials/running-varnish-docker/#setting-the-cache-size